### ISSUE TITLE
Dataset split problem in exchange rate

### ISSUE THREAD

**Issue by bangbangDong** [thuml/Time-Series-Library#153]
In actual operation, it was found that when the output length of exchange_rate data set was 720, the sample number of training set was 4496, the sample number of verification set was only 41, and the sample number of test set was 798. Is this data division not very reasonable (the number of verification sets was too small)? Because the size of the sample varies too much in this ratio.

**Comment by wuhaixu2016** [comment]
Yes, the validation set is too small.
I think you can change the ratio of train and val dataset to balance model training and model validation. It is a possible way to  improve the model perfromance.
